src/server_manager/web_app/digitalocean_server.ts

```diff
@@ -307,7 +307,7 @@ class DigitalOceanHost implements server.ManagedServerHost {
   getMonthlyOutboundTransferLimit(): server.DataAmount {
     // Details on the bandwidth limits can be found at
     // https://www.digitalocean.com/community/tutorials/digitalocean-bandwidth-billing-faq
-    return {terabytes: this.dropletInfo.size.transfer};
+    return {terabytes: this.dropletInfo.size.transfer * (2 ** 40)};
```

Comment:
Do not convert. The size.transfer value is already in terabytes.

Reply:
It is not in terabytes (TB), it's in terabibytes (TiB).

Reply:
Thanks